roles/letsencrypt/tasks/dns-challenge.yml

```diff
@@ -47,4 +48,10 @@
   include_tasks: dns-challenge-hetzner.yml
   when:
     - letsencrypt_do_dns_challenge | bool
-    - letsencrypt_dns_provider == "hetzner"
\ No newline at end of file
+    - letsencrypt_dns_provider == "hetzner"
+
+- name: do dns challenge for testing
+  include_tasks: dns-challenge-test.yml
```

Comment:
hmm, I#m undecided ... calling it "pebble" may be a better approach. What do you think?

Reply:
yeah, sounds better.